src/runtime/Python.Runtime.csproj

```diff
@@ -92,6 +92,7 @@
     <Optimize>false</Optimize>
     <DebugType>full</DebugType>
     <PlatformTarget>x64</PlatformTarget>
+    <DefineConstants>TRACE;DEBUG;PYTHON3;PYTHON35;UCS2</DefineConstants>
```

Comment:
this gets overwritten by setup.py or specific needs of the user, so no need to update this